app/database/server/models/team.ts

```diff
@@ -87,7 +87,7 @@ export default class Team extends Model {
     /** groupsInTeam : All the groups associated with this team */
     @children(GROUPS_IN_TEAM) groupsInTeam!: GroupsInTeam[];
 
-    /** myTeam : Lazy query property returning only the team member that this user is part of  */
+    /** myTeam : Lazy query property returning only the teams that this user is part of  */
```

Comment:
if this is plural, then the property should be `myTeams` ?

Reply:
Yes it should; (1 or more teams). Corrected.

Reply:
well in reality this should be one, basically I can be a member of the team or not. The lazy query should return only one value or null/undefined.

Reply:
Thanks @enahum .  I lost track yesterday.  Team and MyTeam share a 1:1 relationship.   I've updated the comments to specify that the query result might be empty/blank if the user is not part of any team.